PI-System-Deployment-Samples/OnPrem/Verify-PIServer.ps1

```diff
@@ -0,0 +1,40 @@
+# Verify test install completed successfully
+$sqlservice = Get-Service -DisplayName "SQL Server (SQLExpress)"
```

Comment:
whats this file used for?  maybe need a word in the readme that you can use this one to test if the isntall was successful

Reply:
Added a "Verify" section to the README describing that script, and an extra comment in the script itself describing what it does.